src/test/java/racingcar/util/RandomNumberGeneratorTest.java

```diff
@@ -0,0 +1,19 @@
+package racingcar.util;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class RandomNumberGeneratorTest {
+    private static final int TESTCASE_NUM = 10000;
+
+    @Test
+    @DisplayName("랜덤 숫자 생성")
+    void generateRandomNumber() {
+        final NumberGenerator randomNumber = new RandomNumberGenerator();
+        for (int i = 0; i < TESTCASE_NUM; i++) {
+            assertThat(randomNumber.generate()).isBetween(0, 9);
+        }
+    }
+}
```

Comment:
```suggestion
}

```

혹시 10000번을 돌린 이유가 있나요?

Reply:
사실 랜덤 값이 0~9 사이라는 테스트를 어떻게 돌려야할 지 모르겠어서 그냥 최대한 많은 빈도 수로 확인해보자는 의미가 컸는데, 이는 확률적인 테스트이기 때문에 좋은 테스트인가? 라는 생각이 들었습니다. 인터넷에 다른 글을 참고해본 결과 랜덤 생성이라는 테스트는 제공되는 라이브러리를 테스트 하겠다는 것이기 때문에 라이브러리를 믿고 해당 테스트를 진행하지 않아야겠다고 결론을 지었습니다.... 

혹시 다른 의미로 물어보신거라면 힌트를 남겨주시면 더 고민해보겠습니다!!

Reply:
네 정답을 잘 찾아가셨네요 :) 
라이브러리를 극단적으로 할 필요가 없다는 의미였습니다.
그러나 이건 정책이 들어가있죠? 현재 같은 경우에는 10이하의 값이 나오는 부분일거 같아요. 
누군가 이 정책을 깰수도 있으니 테스트 코드는 넣어두셔야 합니다 :)